.github/workflows/integration-build-product.yml

```diff
@@ -95,6 +102,7 @@ env:
   SLACK_CHANNEL: ${{ inputs.overwrite_slack_channel }}
   # development slack channel
   #SLACK_CHANNEL: "C05K2KF1UP8"
+  PROJECT_DATA_VIEWER_API: "https://personal-7vf0v2cu.outsystemscloud.com/ProjectDataViewer5/rest/V1//CreateProject"
```

Comment:
should we create a DNS alias for this ?

Reply:
I not knowledgable in this topic. Is it good to do it in this scenario?

Reply:
Not a bad idea. Specially because the app is on a temporary in a server.

Reply:
@AlexFernandes-MOVAI It's ready from my side. Do we have to do the DNS thing?

Reply:
not blocking